src/app/views/schedule/Schedule.scss

```diff
@@ -51,7 +50,7 @@
     .schedule-countdown {
       width:55vw;
       text-align: left;
-      font-family: "Chakra Petch", sans-serif;
+      font-family: $fontFamilyBase, serif;
```

Comment:
It should be safe to remove these extraneous `font-family` styles since no part of the page has a different font from the main one.

Reply:
Sorry, I should have done that in the first place. I updated the codebase to reflect this.

Reply:
No need to apologize, you probably didn't know; that's part of why we have these reviews.